### PR TITLE
added CSS for pretty much every page, also had to amend some view fil…

### PR DESCRIPTION
**Pull request by lisaoausb**
Added in CSS for pretty much every page, changed some views to add in classes/divs for grid blocks, some more styling might be needed on the friendship page, as I could not fully see what was going on there.

### CODE REVIEW

**Review by adrianHards — COMMENTED** [review]
Some instances of using same classes on nested elements (see comments). Indentation also needs to be fixed for readability. Going through conflicts now. Some changes to forms that are breaking (need to keep enctype="multipart/form-data") for example.

File: views/sessions/new.hbs [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+<h1>Sign In</h1>
+<div class="container">
 <form id="new-session-form" action="/sessions" method="post" class="p-5">
-  <div class="form-group">
+  <div class="item form-group">
     <label for="email">Email:</label>
     <input type="email" class="form-control" id="email" name="email" value="" placeholder="Enter email" required>
   </div>
-  <div class="form-group">
+  <div class="item form-group">
```
Comment (adrianHards) [inline views/sessions/new.hbs:12]: welcome to use old class form-group, I feel like it's more meaningful than item. Or change to form-item?

File: views/posts/index.hbs [hunk 2]
```diff
@@ -1,6 +1,6 @@
+<div class="feed-container">
 
-
-{{#if messages.success}}
+<div class="messages">{{#if messages.success}}
```
[the remark this answers is not in the document]
Comment (lisaoausb) [reply views/posts/index.hbs:3]: yes! now removed nested element because they don't need to be lists. thanks!

**Comment by lisaoausb** [comment]
> Some instances of using same classes on nested elements (see comments). Indentation also needs to be fixed for readability. Going through conflicts now. Some changes to forms that are breaking (need to keep enctype="multipart/form-data") for example. Other than that, nice work.
> 
> Edit: run `npx prettier --check .` to see html issues

Thanks, Adrian! I think indentation should now be fixed and I addressed the other comments. Not sure what enctype is – I cmd + F the files changed and could not find, I also don't remember it. Maybe you can point me there at 2.

**Pull request closed without merging**